install/hls-install.cabal

```diff
@@ -19,7 +19,7 @@ library
                      , Env
                      , Help
   build-depends:       base >= 4.9 && < 5
-                     , shake >= 0.16.4 && < 0.19
+                     , shake >=0.17.5
```

Comment:
```suggestion
                     , shake >= 0.17.5
```

We usually have a space after `>=`

Reply:
Okay... but it seems that `cabal-fmt` will remove this space

Reply:
Meh, if there's a formatter removing it, don't worry. Although it feels awfully inconsistent that it removes the space here and adds it on the line above. This is a very light suggested change, and if it's not the right thing, don't bother.

Reply:
Ah, I see. It's inconsistent because I copied  `shake >=0.17.5` from another cabal file in our codebase, whereas this cabal file haven't gotten formatted yet.